.github/workflows/trigger-preview.yaml

```
@@ -2,24 +2,22 @@ name: trigger-preview
 on:
   workflow_run:
     workflows:
-      - trigger-site-build
+      - trigger-book-build
     types:
       - requested
       - completed
 
 jobs:
   find-pull-request:
     uses: ProjectPythia/cookbook-actions/.github/workflows/find-pull-request.yaml@main
-
   deploy-preview:
     needs: find-pull-request
     if: github.event.workflow_run.conclusion == 'success'
     uses: ProjectPythia/cookbook-actions/.github/workflows/deploy-book.yaml@main
     with:
       artifact_name: book-zip-${{ needs.find-pull-request.outputs.number }}
-      destination_dir: _preview/${{ needs.find-pull-request.outputs.number }}  # deploy to subdirectory labeled with PR number
-      is_preview: 'true'
-      publish_dir: 'portal/_build/html'
+      destination_dir: _preview/${{ needs.find-pull-request.outputs.number }} # deploy to subdirectory labeled with PR number
+      is_preview: "true"
```

I believe this line should still be here:
```suggestion
      is_preview: "true"
      publish_dir: 'portal/_build/html'
```